src/styles/blocks/buttom.scss

```
@@ -0,0 +1,29 @@
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
```

No need to hardcode width
```suggestion
  width: 100%;
```